package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "xendit-node",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
No need to bump, I will bump during the release process with `np`

Reply:
Updated @stanleynguyen